sockeye/encoder.py

```diff
@@ -761,6 +782,40 @@ def append(self, cls, infer_hidden: bool = False, **kwargs) -> Encoder:
         return encoder
 
 
+class EmptyEncoder(Encoder):
+    """
+    This encoder doing nothing.
+    :param config: configuration.
+    """
+
+    def __init__(self,
+                 config: EmptyEncoderConfig) -> None:
+        super().__init__(config.dtype)
+        self.num_embed = config.num_embed
+        self.num_hidden = config.num_hidden
+
+    def encode(self,
+               data: mx.sym.Symbol,
+               data_length: Optional[mx.sym.Symbol],
+               seq_len: int) -> Tuple[mx.sym.Symbol, mx.sym.Symbol, int]:
+        """
+        Encodes data given sequence lengths of individual examples and maximum sequence length.
+        :param data: Input data.
+        :param data_length: Vector with sequence lengths.
+        :param seq_len: Maximum sequence length.
+        :return: Encoded versions of input data with all-zero-values.
```

Comment:
Since this isn't actually any representation of the input, it could just be: "Expected number of empty states (zero-filled)."